include/pybind11/detail/internals.h

```diff
@@ -314,16 +314,45 @@ inline internals **&get_internals_pp() {
     return internals_pp;
 }
 
-// forward decl
-inline void translate_exception(std::exception_ptr);
+const std::forward_list<ExceptionTranslator> &get_exception_translators();
+const std::forward_list<ExceptionTranslator> &get_local_exception_translators();
+
+// Apply all the extensions translators from a list
```

Comment:
Is `extensions` correct here? I see this comment is just moved from pybind11.h, but maybe it's a long-standing oversight?